### PR TITLE
Trivial cleanups and one bugfix

### PR DESCRIPTION
**Pull request by agriffis**
These are mostly noop cleanups except for one actual bugfix handling before_record_response.

### CODE REVIEW

**Comment by kevin1024** [comment]
Awesome!  Thanks for the cleanup!  :cake:

Could you add a test for the bug that you fixed with `before_record_response`?  Otherwise this is fantastic!

**Comment by agriffis** [comment]
There you go. Test is included in the commit with the change, i.e. 3e5553c

File: vcr/errors.py [hunk 1]
```diff
@@ -3,8 +3,7 @@ class CannotOverwriteExistingCassetteException(Exception):
 
 
 class UnhandledHTTPRequestError(KeyError):
-    '''
-    Raised when a cassette does not c
-    ontain the request we want
-    '''
+    """
+    Raised when a cassette does not contain the request we want
```
Comment (IvanMalison) [inline vcr/errors.py]: We've been trying to make this more python standard compliant:
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

**Comment by agriffis** [comment]
@IvanMalison well, I barely got that fixed before you merged it. ;-)

**Comment by IvanMalison** [comment]
hah sorry i just realized ti would be easier for me to fix it. Thanks for sneaking it in though

**Pull request merged**